food-for-thought/app/map.tsx

```diff
@@ -39,14 +45,14 @@ export type Restaurant = {
     icon: string
   }[],
   // price rating out of 1: cheap, 2: average, 3: expensive, 4: very expensive 
-  price?: number,
+  price: number,
   // rating out of 10
-  rating?: number,
-  total_ratings?: number,
-  menuId?: string,
+  rating: number,
+  total_ratings: number,
+  menuId: string,
   restaurantPhotos?: string[],
   foodPhotos?: string[],
-  hasMenu?: boolean
+  hasMenu: boolean
```

Comment:
Just curious, are we making these non-nullable cause they are being defined elsewhere and can't be null? Just being cautious cause we may get undefined errors later down the line

Reply:
I changed these to reflect the `required` fields in the Restaurant Schema